File: 프로그래머스/1/131705. 삼총사/삼총사.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> number) {
    int answer = 0;
    
    int length = number.size();
    for (int x = 0; x < length - 2; x++) {
        for (int y = x + 1; y < length - 1; y++) {
            for (int z = y + 1; z < length; z++){
                if (number[x] + number[y] + number[z] == 0) {
                    answer++;
                }
            }
        }
    }
    
    return answer;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    answer = 0
    length = len(number)
    
    for x in range(length - 2):
        for y in range(x + 1, length - 1):
            for z in range(y + 1, length):
                if number[x] + number[y] + number[z] == 0:
                    answer += 1
    return answer